cmd/consensus/main.go

```diff
@@ -188,7 +188,7 @@ func main() {
 				assigner,
 				requireOneApproval,
 			)
-			requesterEng.WithHandle(match.HandleReceipt)
+			requesterEng.WithHandle(match.HandleReceipt) // TODO: use match.Process here (?) to parallelize engines in terms of threading
```

Comment:
`match.HandleReceipt` internally is using [match.process already](https://github.com/onflow/flow-go/blob/5478e5db1dc160f9c8f069edce2da04dd2ea180e/engine/consensus/matching/engine.go#L174)

Reply:
thanks good point. I meant that we could spin out the `match.process` call into its own go routine. You are right, `match.Process` does not achieve this, as it uses `unit.Do`.